src/auction.js

```diff
@@ -642,47 +643,51 @@ export const getAdvertiserDomain = () => {
   }
 }
 
+// factory for key value objs
+function createKeyVal(key, value) {
+  return {
+    key,
+    val: (typeof value === 'function')
+      ? function (bidResponse, bidReq) {
+        return value(bidResponse, bidReq);
+      }
+      : function (bidResponse) {
+        return getValue(bidResponse, value);
+      }
+  };
+}
+
+const defaultAdserverTargeting = cache(() => {
```

Comment:
Both `getPriceByGranularity` and `getAdvertiserDomain` should return equivalent closures no matter what the config is (the return value is what looks it up) so caching should not be a problem.

But I do think there's a good argument to be made for not caching - I am doing it here because it was in the original code, which over time got broken because more stuff was added in a non-cacheable way (in `getStandardBidderSettings`). 

So now we still have to recalculate the "standard" settings for each auction, plus we have to be mindful of what is being cached to avoid making unintentional changes to global state. I doubt the added complexity is worth the tiny performance improvement, so I also vote for removing the cache.